public/components/player/list/player_list_test.html

```diff
@@ -0,0 +1,2 @@
+<script src="../../../node_modules/steal/steal.js" main="bitballs/components/player/list/player_list_test"></script>
```

Comment:
normally, we just call this `test.html`.  We include "player_list" in `player_list_test.js` because having a bunch of `test.js`'s open would be annoying in most ides.  However, we don't typically have to open the `test.html` files once they are created.